config/locales/en.yml

```diff
@@ -33,12 +33,12 @@ en:
     url_applying: https://getintoteaching.education.gov.uk/steps-to-become-a-teacher#step-5
     url_get_an_adviser: https://adviser-getintoteaching.education.gov.uk
     url_get_an_adviser_start: https://getintoteaching.education.gov.uk/teacher-training-advisers
-    url_international_candidates: https://getintoteaching.education.gov.uk/international-candidates
-    url_training_with_a_disability: https://getintoteaching.education.gov.uk/guidance/become-a-teacher-in-england#training-to-teach-if-you-have-a-disability
+    url_international_candidates: https://getintoteaching.education.gov.uk/non-uk-teachers
```

Comment:
This will 404 until GIT ship this new page. This is why it's in draft.

Reply:
Ah I thought those were all live, I wonder if we kept this ticket in backlog and moved to the dev column when this was ready for development and all URLs were confirmed and live, to avoid having to wait on this. Let me know your thoughts

Reply:
They're shipping it today, so not too much to wait on.